### PR TITLE
Added version healthcheck server, incl. tests

### PR DESCRIPTION
**Pull request by alexbasalyga**
About the naming choice: so 'require' wouldn't confuse the version-healthcheck npm package with the local version healthcheck server, and because 'version-healthcheck-server' is needlessly wordy, the local module is called 'version-healthchecker' wherever it is referenced (whether in a variable or file name, etc.). Let me know if you want me to refactor to another name.

I'll leave the version number bump to you in case this pull request isn't closed immediately.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
     "validate": "npm run lint && npm run build && npm run test && npm run functional && npm outdated --depth 0",
     "validate-dev": "npm run lint && npm run build && npm outdated && npm run test-dev",
     "watch:build": "watch --wait=5 'npm run validate-dev' ./app/",
-    "dev": "npm run watch:build"
+    "dev": "npm run watch:build",
+    "version-healthchecker": "npm run build && node app/version-healthchecker.js"
```
Comment (ericelliott) [inline package.json]: We shouldn't run the build in this step. Best practice is to build and validate the build prior to promoting the app to production, so it doesn't make sense to combine the build step and the app start unless you're working on the development.

**Comment by ericelliott** [comment]
I think `healthcheck` would be a less awkward name for this.

**Comment by ericelliott** [comment]
And thank you! =) :+1:

**Comment by alexbasalyga** [comment]
Fixed.

File: app/test/unit/healthcheck/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+var
+  test = require('tape'),
+  supertest = require('supertest'),
+  app = require('healthcheck'),
+  arrayIntersect = require('array-intersection'),
+  fs = require('fs');
+
+var 
+  pkg = require(__dirname + '/../../../../package.json'),
```
Comment (ericelliott) [inline app/test/unit/healthcheck/index.js]: Is there a good reason not to use `rootrequire` here? https://github.com/ericelliott/isomorphic-express-boilerplate#requiring-modules

**Comment by alexbasalyga** [comment]
An oversight on my end. I'll fix it.

**Comment by alexbasalyga** [comment]
Alright. This should be ready to merge now. :)

**Comment by ericelliott** [comment]
Sorry, I've been very busy. Please remind me if I don't merge this in the next few days. =)

**Comment by alexbasalyga** [comment]
Bump :+1:

**Comment by ericelliott** [comment]
Thanks... busy time continues until early next week, probably.

File: app/node_modules/healthcheck/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+// This route prepends the cache-control
+// middleware so that the browser always logs
+// to the server instead of fetching a useless
+// OK message from its cache.
+// 
+// Using a 1x1 transparent gif allows you to
+// use the logger in emails or embed the tracking
+// pixel on third party sites without resorting
+// to JavaScript.
+app.get( '/log.gif', noCache, log.route() );
```
Comment (ericelliott) [inline app/node_modules/healthcheck/index.js]: This isn't needed for this service.

File: package.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
     "validate-dev": "npm run lint && npm run build && npm outdated && npm run test-dev",
     "watch:build": "watch --wait=5 'npm run validate-dev' ./app/",
     "dev": "npm run watch:build",
+    "healthcheck": "node app/healthcheck.js",
```
Comment (ericelliott) [inline package.json]: We should probably call this `start-healthcheck` and we should add `npm run start-healthcheck` to the `start` script, as well.

**Comment by ericelliott** [comment]
Again, thanks for pitching in. Great job over-all. Just a couple comments. =)

**Comment by alexbasalyga** [comment]
Fixed. :+1:

**Pull request merged**